server/backend/src/utils/auth/tokenAuthoriser/Index.ts

```diff
@@ -8,9 +8,9 @@ import * as HttpResponse from '@karya/http-response';
 const policyParser = new PolicyParser(Policy);
 
 export const tokenAuthoriser: UserRouteMiddleware = async (ctx, next) => {
-  const resourceTokens = policyParser.getResourceTokens(ctx);
+  const resourceTokens = policyParser.getResourceTokens(ctx); //reource == api
   const serverUser = await BasicModel.getSingle('server_user', { id: ctx.state.entity.id });
-  const userTokens = await TokenAuthHandler.getTokens(serverUser);
+  const userTokens = [serverUser.role];
```

Comment:
Line 13: You have to use `serverUser.role_mappings` here instead of `[serverUser.role]`